app/models/book.rb

```diff
@@ -17,6 +17,7 @@ class Book < ApplicationRecord
     joins(:author)
       .where("authors.name LIKE :q OR books.name LIKE :q", q: "%#{search}%")
   end)
+  scope :find_books, ->{where id: session[:book_ids]}
```

Comment:
em truyền tham số vào cho linh hoạt để sử dụng
```suggestion
  scope :find_books, ->(ids){where id: ids}
```